Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -230,6 +254,18 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output, expected)
     }
 
+    func testMaxPool3DGradient(){
+        let layer = MaxPool3D<Float>(poolSize: (2, 2, 2), strides: (1, 1, 1), padding: .valid)
+        let input = Tensor(shape: [1, 2, 2, 2, 1], scalars: (0..<8).map(Float.init))
+        let computedGradient = gradient(at: input, layer) { $1($0).sum() }
+        let expectedGradient = Tensor<Float>([[
```

Comment:
```suggestion
        let expectedGradient =  Tensor<Float>(shape: [1, 2, 2, 2, 1),
                                              scalars: [0, 0, 0, 0, 0, 0, 0, 1])
 ```

Reply:
Just a suggestion for better readability.

Reply:
I was going to do that but it seemed easier to visualize which indices carry gradient information from above.